app/domain/authentication/authn_jwt/jwt_configuration_factory.rb

```diff
@@ -0,0 +1,17 @@
+module Authentication
+  module AuthnJwt
+    # Factory that receives vendor name and returns appropriate JWT vendor configuration class
+    class JwtConfigurationFactory
+      VENDORS = {
+        "dummy" => JWTConfigurationDummyVendor
+      }
+
+      def get_jwt_configuration(vendor)
+        unless VENDORS.key?(vendor)
+          raise "Vendor #{vendor} not implemented yet."
```

Comment:
feel free to do this in another PR later on but verify that you have a task to convert all errors to Conjur ones (`CONJ...`)

Reply:
can you create an issue for this?